Integrations/Tenant True Scripts/IR-v4.31.0-and-later/tenant_config_migration_security_classifications.py

```diff
@@ -16,63 +14,127 @@
 post_apitoken = config.post_apitoken
 post_head = config.post_head

```

Comment:
Wasn't sure wether to change the naming on these/what a typical config file might look like